pgxn/neon/pagestore_smgr.c

```diff
@@ -2592,3 +2642,130 @@ smgr_init_neon(void)
 	smgr_init_standard();
 	neon_init();
 }
+
+
+/*
+ * Return whether we can skip the redo for this block.
+ * 
+ * The conditions for skipping the IO are:
+ *
+ * - The block is not in the shared buffers
+ * - The block is not in the file cache (or is evicted from that cache)
+ * 
+ * It is important to note that skipping WAL redo for a page also means
+ * the page isn't locked by the redo process, as there is no Buffer
+ * being returned, nor is there a buffer descriptor to lock.
+ * This means that any IO that wants to read this block needs to wait
+ * for the WAL REDO process to finish before it allows the system to start
```

Comment:
instead of `wait for the WAL REDO process to finish`
should it be `wait for the WAL REDO process to reach read lsn` ?

because wal redo for hot standby never finishes